internal/doculint/doculint.go

```diff
@@ -104,14 +104,14 @@ func init() { //nolint:gochecknoinits // Why: This is necessary to grab flags.
 
 // doculint is the function that gets passed to the Analyzer which runs the actual
 // analysis for the doculint linter on a set of files.
-func doculint(pass *analysis.Pass) (interface{}, error) { //nolint:funlen // Why: Doesn't make sense to break this function up anymore.
+func doculint(_pass *analysis.Pass) (interface{}, error) { //nolint:funlen // Why: Doesn't make sense to break this function up anymore.
```

Comment:
This rename is terrible. Just call them all `pass` and reassign your function argument at that point. This isn't adding any value.

Reply:
we can't reassign `pass` because the type returned from `reporter.NewPass` is an interface that only captures a subset of the functionality that `analysis.Pass` provides